src/Symfony/Component/Validator/Password/PasswordStrengthEstimatorInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Password;
+
+use Symfony\Component\Validator\Constraints\PasswordStrength;
+
+interface PasswordStrengthEstimatorInterface
+{
+    /**
+     * Returns the estimated strength of a password.
+     *
+     * The higher the value, the stronger the password.
+     *
+     * @return PasswordStrength::STRENGTH_*
+     */
+    public function estimateStrength(#[\SensitiveParameter] string|\Stringable $password): int;
```

Comment:
Do we expect password estimators to handle stringables differently? If not, I'd narrow the interface to strings only.

```suggestion
    public function estimateStrength(#[\SensitiveParameter] string $password): int;
```

Reply:
Indeed => the Stringable is on the Validator but can be removed from the estimator